From 1285b8b36ebc22dd11359a936a670e34e7fceeec Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Wed, 15 Nov 2023 08:25:11 +0000
Subject: [PATCH 0724/1085] overlays: imx296: Fix cam port override for
 regulators

The override was missing/incorrect for the regulator labels.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/imx296-overlay.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/imx296-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx296-overlay.dts
@@ -97,8 +97,9 @@
 		cam0 = <&i2c_frag>, "target:0=",<&i2c_csi_dsi0>,
 		       <&csi_frag>, "target:0=",<&csi0>,
 		       <&clk_frag>, "target:0=",<&cam0_clk>,
+		       <&reg_frag>, "target:0=",<&cam0_reg>,
 		       <&imx296>, "clocks:0=",<&cam0_clk>,
-		       <&imx296>, "VANA-supply:0=",<&cam0_reg>;
+		       <&imx296>, "avdd-supply:0=",<&cam0_reg>;
 		clock-frequency = <&clk_over>, "clock-frequency:0";
 	};
 };
